File: wrappers/mvc/demos/Kendo.Mvc.Examples/Controllers/Pie_Charts/Local_Data_BindingController.cs
using System;
using System.Linq;
using System.Web.Mvc;
using Kendo.Mvc.Examples.Models;

namespace Kendo.Mvc.Examples.Controllers
{
    public partial class Pie_ChartsController : Controller
    {
        public ActionResult Local_Data_Binding()
        {
            return View(ChartDataRepository.SpainElectricityBreakdown());
        }
    }
}

File: wrappers/java/spring-demos/src/main/java/com/kendoui/spring/controllers/piecharts/LocalDataBindingController.java
package com.kendoui.spring.controllers.piecharts;

import org.springframework.stereotype.Controller;
import org.springframework.ui.Model;
import org.springframework.web.bind.annotation.RequestMapping;
import org.springframework.web.bind.annotation.RequestMethod;

import com.kendoui.spring.models.ChartDataRepository;

@Controller("dataviz-pie_chart-local_data-controller")
@RequestMapping(value="/pie-charts/")
public class LocalDataBindingController {
    @RequestMapping(value = "/local-data-binding", method = RequestMethod.GET)
    public String index(Model model) {
        model.addAttribute("spainElectricityBreakdown", ChartDataRepository.SpainElectricityBreakdown());
     
        return "/pie-charts/local-data-binding";
    }
}
